### ISSUE TITLE
Image not being taken or downloaded

### ISSUE THREAD

**Issue by orfest** [mworion/MountWizzard4#112]
Potentially a duplicate of #107 

I'm running Ekos on ubuntu which starts an indi server and connects to the camera (Nikon D7000).
Then I'm starting MW4 which connects to the camera without any issues.
If I take an image in Ekos, MW4 notices that and displays messages next to the green Camera indicator: "exposure" and then "downloading". Taking image in Ekos works.
However, if I press "Expose 1" in the "Image" window in MW4, then nothing happens.

MW4 v0.220.3, I failed to save a log.
Updating to v1.0.3 right now, will try again tomorrow and will make sure to save the log.

**Comment by orfest** [comment]
Upgraded to v1.0.3, but still doesn't work.
Log: https://dpaste.com/HEDNMHYAR

Thank you very much for looking at the issue!

**Comment by mworion** [comment]
ortest,
could you change loglevel to debug_trace and test again ? The log file could be dropped here in Github easily. An I could keep them for revise.
I do not have that much experience with DSLR's, so my guess is the debayering might cause the issue.
Michel

**Comment by orfest** [comment]
My guess is that MW4 uses BULB exposure to take photos.
I've noticed that I can't use BULB exposures with Ekos at all.

For example, if I set "Force Bulb = Off", can take 1s photos.
If I set "Force Bulb = On", then 1s photos do nothing. The shutter doesn't click. Ekos starts counting down the time, but the camera completely ignores that command.

If my guess is correct, then the issue isn't with MW4, but with my Ekos<->Camera interaction in the Bulb mode.
